src/test/java/ReversTest.java

```diff
@@ -0,0 +1,78 @@
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class ReversTest {
+
+    @Test
+    public void keysEqualsValues() throws Exception {
```

Comment:
This Exception isn't thrown in the method - please remove it from the method signature

Reply:
Done